website/hw/hw5_rubric.rst

```diff
@@ -0,0 +1,50 @@
+Homework 5 Rubric
+=================
+
+The grading of this project follows a specifications grading approach. If you have not already
+done so, make sure to read our `Grading <../grading.html>`__ page for more details.
+
+When grading your work, the graders will be checking for the following issues in your code. Please
+note that some of the issues are labelled as "[Major]" (these will have a greater effect on your score
+for the homework)
+
+Task 1
+------
+
+- Did not write tests for ``segment_new``, ``segment_init``, or ``segment_free``
```

Comment:
Is there a reason that this is not classified as a major issue, while not implementing tests for `on_segment`` or ``point_orientation`` is a major issue?

Reply:
Because these are pretty straightforward to write (and amount to writing tests that are very similar to the existing new/init/free tests for point and polygon). The on_segment and point_orientation tests actually require thinking harder about coverage, etc.